intake_esm/aggregate.py

```diff
@@ -123,10 +157,12 @@ def concat_time_levels(dsets, time_coord_name_default, restore_non_dim_coords=Fa
             return dsets[0]
 
     attrs = dict_union(*[ds.attrs for ds in dsets])
+    time_coord_name = ensure_time_coord_name(dsets[0], time_coord_name_default)
 
+    # Equivalent to xr.align(*dsets, tolerance=xtol, exclude='time')
+    dsets = _override_coords(dsets, time_coord_name)
```

Comment:
We had discussed making this optional. Should it be optional?

Reply:
We probably shouldn't be silent about this behavior. I went ahead and made it optional, allowing users to opt-in or opt-out..